Jacdac extension (#5)
KitronikLtd/pxt-kitronik-air-quality: jacdac/README.md

# Jacdac Kitronik Air Quality library

> The functionality of the board are exposed as Jacdac servers instead of native blocs, so that the user can leverage the jacdac simulation in MakeCode. The Jacdac server are not enabled in the simulator and turned on when running on hardware.

Reviewer: ```suggestion
The functionality of the board is exposed as Jacdac servers instead of native blocks so that the user can leverage the jacdac simulation in MakeCode. The Jacdac servers are not enabled in the simulator but run on hardware.
```
